core/modules/system/system.install

```diff
@@ -3523,6 +3545,31 @@ function system_update_1094() {
   $config->save();
 }
 
+/**
+ * Creates database tables and config items for cache tags if they do not exist.
+ */
+function system_update_1095() {
+  $schema = system_schema();
+
+  if (!db_table_exists('invalidated_cache_tags')) {
```

Comment:
Either this table needs to be created in `system_update_1000()` or there could be a check in `system_do_invalidate_cache_tags()` for whether the table exists before trying to update the table.

Reply:
I added a `db_table_exists()` in `system_do_invalidate_cache_tags()` so this should be fixed now.